models/GraphicsContextSettings/GraphicsContextSettings.js

```diff
@@ -37,6 +39,9 @@ class GraphicsContextSettings {
       Object.assign(this, json);
     } else {
       Object.assign(this, GraphicsContextSettings.Model, args);
+      Object.assign(this, {
```

Comment:
I think without the `GraphicsContextSettings.Model` this won't get the `_class` attribute.

Reply:
Line 41 (https://github.com/amzn/sketch-constructor/pull/67/files#diff-9147adf669b4522b117eacb792424815R41) still assigns the Model, or am I missing something? The we just overwrite the blendMode with the values from the map.

Reply:
Oh duh... will do that, thanks for the hint.

Reply:
@dbanksdesign done :)